src/main/java/io/openliberty/tools/intellij/actions/ViewIntegrationTestReport.java

```diff
@@ -48,17 +49,23 @@ protected void executeLibertyAction(LibertyModule libertyModule) {
 
         // get path to project folder
         final VirtualFile parentFile = buildFile.getParent();
-        File failsafeReportFile = Paths.get(parentFile.getPath(), "target", "site", "failsafe-report.html").normalize().toAbsolutePath().toFile();
-        VirtualFile failsafeReportVirtualFile = LocalFileSystem.getInstance().findFileByIoFile(failsafeReportFile);
 
+        // Dev mode runs the tests and it may have selected a report generator that uses one location or another depending on the version number
+        // Maven plugin maven-surefire-report-plugin v3.5 and above use this location
```

Comment:
suggestion to mention that the name of the file is also different

Reply:
ok